test/model/binding.dart

```diff
@@ -474,27 +474,23 @@ class FakeFlutterLocalNotificationsPlugin extends Fake implements FlutterLocalNo
 }
 
 class FakeAndroidFlutterLocalNotificationsPlugin extends Fake implements AndroidFlutterLocalNotificationsPlugin {
+}
+
+class FakeIOSFlutterLocalNotificationsPlugin extends Fake implements IOSFlutterLocalNotificationsPlugin {
+}
```

Comment:
Neat!

Does this mean we can also cut out the test binding's `resolvePlatformSpecificImplementation`? That'd be a nice further cleanup commit to come after this one.